### PR TITLE
babel-polyfill dependency

### PR DESCRIPTION
**Pull request by alirezamdv**
## Summary

in This PR babel-polyfill replaced with recommendations from babel!
https://babeljs.io/docs/en/babel-polyfill

### CODE REVIEW

File: template/src/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import "babel-polyfill";
+import "core-js/stable";
+import "regenerator-runtime/runtime";
```
Comment (JRJurman) [inline template/src/index.js]: do we not also need `regenerator-runtime`?

**Comment by JRJurman** [comment]
Can you verify that async-await works? That was the main motivation for including these, and now that the dependencies are correct it should work.

**Comment by alirezamdv** [comment]
> Can you verify that async-await works? That was the main motivation for including these, and now that the dependencies are correct it should work.
Hi,
the dependencies are exactly like documentation and should actually work..i didn't understand how could i test it with async, its nice if you could test it or just give me some help to do that..
Thanks

**Comment by JRJurman** [comment]
@alirezamdv sure! here's some code you can include in the generated app.

Add the following lines in the `ColorHeader.js`, right after the `updateOnTitleClick`.
(You'll also need to include the `useEffect` hook in the imports from `tram-one`)

```javascript
  useEffect(async () => {
    const page = await fetch('/')
    const content = await page.text()
    console.log(content)
  })
```

When running the example, the console should print out the following:
image

**Comment by alirezamdv** [comment]
@JRJurman   Thanks for the help, I tested and it works well ...


tram-one

**Comment by JRJurman** [comment]
@alirezamdv awesome! did you ever verify if we needed `regenerator-runtime` in our dependencies list?

**Comment by alirezamdv** [comment]
> @alirezamdv awesome! did you ever verify if we needed `regenerator-runtime` in our dependencies list?


yes i did, we need it, without it we get this error "regeneratorRuntime is not defined
    at Object.parcelRequire.index.js.core-js/stable"

**Comment by JRJurman** [comment]
@alirezamdv if that's the case, can you include that in the `package.json`? Right now it looks like you only added `corejs` was added there.

**Comment by alirezamdv** [comment]
> @alirezamdv if that's the case, can you include that in the `package.json`? Right now it looks like you only added `corejs` there.
> 
> Please update the Summary with the work done, you can look at other closed PRs for an example of what that should look like.


there is no need to to include it in package.json...it comes with core-js 3.3

**Comment by alirezamdv** [comment]
## Summary
in This PR babel-polyfill  replaced with recommendations from babel!
https://babeljs.io/docs/en/babel-polyfill

**Comment by JRJurman** [comment]
@alirezamdv sorry, is there some docs or proof you can show that regenerator-runtime is included with corejs? I'm slightly suspicious, and there's nothing in npm that indicates that regenator-runtime is included in corejs... I might just be oblivious though :D

**Comment by alirezamdv** [comment]
> regenerator-runtime

yes there is a good doc , please take a look at this release note for  [core-js 3](https://babeljs.io/blog/2019/03/19/7.4.0) 😃 

>When using core-js 3, the useBuiltIns: "entry" option not only transforms import "core-js" imports, but also regenerator-runtime/runtime and all the nested core-js entry points.

**Pull request merged**